ui/apps/platform/src/services/GroupsService.ts

```diff
@@ -8,7 +9,7 @@ const updateUrl = '/v1/groupsbatch';
  *
  * @returns {Promise<Object, Error>} fulfilled with array of groups
  */
-export function fetchGroups() {
+export function fetchGroups(): Promise<{ response: { groups: Group[] } }> {
```

Comment:
Commenting here, you can delete `returns` from JSDoc comments where TypeScript is now the source of truth.